app/server/beam/tau/src/tau.app.src

```diff
@@ -1,23 +0,0 @@
-%% This is an -*- erlang -*- file.
-{application, tau,
- [{description, "Sonic Pi Tau Server"},
-  {vsn, ""},
-  {modules, [tau_server,
-             tau_server_app,
-             tau_server_sup,
-             osc
-            ]},
-  {registered,[]},
-  {mod,{tau_server_app,[]}},
-  {env, [{cues_on, true},
-         {midi_on, true},
-         {link_on, true},
-         {osc_in_udp_port, 4560},    % sane defaults for the ports
-         {api_port, 51240},
-         {cue_host, {127,0,0,1}},
-         {spider_port, 51235},
-         {osc_in_udp_loopback_restricted, true}
```

Comment:
These defaults do not match the ones in the Tau.Application but I think those defaults have not been used for a while.

Reply:
Yes, Elixir automatically generates one! I have removed the copies!